includes/html/functions.inc.php

```diff
@@ -90,7 +91,7 @@ function bill_permitted($bill_id)
 function port_permitted($port_id, $device_id = null)
 {
     if (! is_numeric($device_id)) {
-        $device_id = get_device_id_by_port_id($port_id);
+        $device_id = Port::find($port_id)->value('device_id');
```

Comment:
The find() does a select * here which is wasteful

Reply:
I think  @Jellyfrog is right with this. In my tests it `where()->value()` needs the half of time to query. Havn't looked at the memory... Anyway I like to get rid of this helper function in the next PRs once the actual are commited and running for some days in the wild - If you are also fine with this of course :-) but step by step...

Reply:
Switch it back then